frontend/vitest.config.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference types="vitest" />
+/// <reference types="vite/client" />
```

Comment:
Is this type reference required? I haven't seen it in any vitest examples anywhere.